app/main.py

```diff
@@ -1,8 +1,33 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(self, comfort_class, clean_mark, brand):
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
+        self.brand = brand
 
 
 class CarWashStation:
-    # write your code here
-    pass
+    def __init__(self, distance_from_city_center, clean_power, clean_mark, average_rating, count_of_ratings):
+        self.distance_from_city_center = distance_from_city_center
+        self.clean_power = clean_power
+        self.clean_mark = clean_mark
+        self.average_rating = average_rating
+        self.count_of_ratings = count_of_ratings
+
+    def serve_cars(self, cars: list):
+        income = 0
+        for car in cars:
+            if car.clean_mark < self.clean_power:
+                income += self.calculate_washing_price(car)
+        return round(income, 1)
```

Comment:
The `serve_cars` method calculates the income from washing cars but does not update the `clean_mark` of the cars that are washed. Consider calling the `wash_single_car` method within this loop to update the `clean_mark` of each car that is washed.